src/mscorlib/src/System/Reflection/AssemblyName.cs

```diff
@@ -533,7 +571,83 @@ internal string GetNameWithPublicKey()
         private extern byte[] nGetPublicKeyToken();
     
         [System.Security.SecurityCritical]  // auto-generated
-        [MethodImplAttribute(MethodImplOptions.InternalCall)]
-        static internal extern String EscapeCodeBase(String codeBase);
+        static internal String EscapeCodeBase(String codebase)
+        {
+            return EscapeString(codebase, false, true, true);
```

Comment:
Here's the code I tried:
```c#
        var an = new AssemblyName();
        an.CodeBase = "#[]";
        Console.WriteLine(an.EscapedCodeBase);
```
I'm getting the output `#[]`. Is there something I'm doing differently?